### DISCUSSION TITLE
Anticipated value driver/obstacles by stakeholder to broader adoption of R for HTA analytics

### DISCUSSION THREAD

**Discussion by agorstras** [RConsortium/HTA-wg#6] [General]
Taking our discussion from 27-Nov-2024 R Consortium HTA-WG meeting offline. 

- Do you agree with the below?
- What are other relevant hypotheses?
- Commonalities among drivers or obstacles?

| Stakeholder | Anticipated driver | Anticipated obstacle |
|---------------|--------------------------|--------------------------------|
| Pharma | Shape more efficient and robust analysis pipelines | Needs substantial investments with questionable return |
| CROs | Improve efficiency and consistency between clients | Increased challenges with differentiation; gained efficiencies may benefit pharma more |
| Academia | Push novel methodology into 'production' more easily]{style="color:green;} | More contraints (new packages having to interact with standard packages) |
| HTA bodies | Increase quality & consistency, reduce assessor burden | Creating unwarranted or difficult-to-control precedents] |

**Comment by jayhpark1** [comment]
Putting on my academic hat and in thinking of the future workforce as an employer, it'd be interesting to assess what the current curriculum health economics, stats, and data science related courses that are offered to our graduate students. Unfortunately, it is likely that Canadian institutions may be falling behind especially compared to UK Universities like University of Sheffield. 

I can volunteer to look into Canadian institutions.

**Comment by dom-muston** [comment]
Thanks for a great discussion today.

We discussed that it may be easier to think about these things if we distinguished different types of HTA Analytics. After all, using R to produce tables of AE risks is quite different to using R for a cost-effectiveness model. Distinction along the lines of 'clinical vs economic' may not work because there are several grey areas (e.g. analyses of health utility, survival modeling of trial time-to-event endpoints).

May I instead propose to differentiate HTA Analytics along the lines of:
* Involves analysis of individual patient-level data (IPD). This would include descriptive trial analyses, survival modeling, mixed effects modeling of health utilities, indirect comparisons using population-adjusted methods. (It may not technically include certain network meta-analyses, but perhaps should.)
* Does not involve analysis of IPD. This would include HTA Analytics such as budget impact and cost-effectiveness economic models, where results of analyses of IPD are taken as inputs.

A benefit of R of course is that a single set of functions/procedures may address both analysis of IPD and consequent economic modeling. However, I would still suggest to separate both parts because typically I think an HTA may wish to verify any IPD analysis before considering economic modeling methods or results. It may also align with Pharma organization or security arrangements, in which access to IPD may be available to a more limited audience - so different functions do different parts of the work.

**Comment by dom-muston** [comment]
More generally, I would love to get on the table as many of the drivers and obstacles that we can think of.

From my perspective, as someone who might personally love to fully move to R tomorrow, there are very legitimate reasons for Pharma and HTAs to continue using Excel for economic models. Pharma needs to be able to adapt economic models easily, maintain consistency between markets, run multiple scenario and sensitivity analyses of different types, and interact with internal and external colleagues in low resource situations, all within often tight timelines. HTAs expect to be able to scrutinise economic models to a similar degree as they are used to with Excel cell-by-cell inspections (despite this being an insufficient and unreliable quality assurance approach). R and its available packages do not, in my opinion, address these needs very well yet - but I'm sure they could! We are also hampered, I think, by the absence of a Common Data Model (or similar) for inputs to health economic models.

If we can get the drivers and obstacles on the table, then we can consider how solutions might be developed. I look forward to reading comments and ideas on this thread.

**Comment by jrdnmdhl** [comment]
Another challenge I'd add on the Pharma side is turnover. It's really hard to drive adoption of new technologies in these organizations when the people within them, including the decision makers, are constantly changing.
